add/index.xql

```
@@ -46,10 +46,54 @@ let $comment := comment{"
 "
 }
 
+let $eoEditionFileNone := <html xmlns="http://www.w3.org/1999/xhtml">
```

there's already a lot of HTML in this file and a lot of duplicated code which makes it hard to maintain. My recommendation would be to move this HTML to an extra folder (e.g. "templates") and try to create some HTML snippets that could be reused by other HTML files via xinclude.

---

I'm absolutely with you! My first try was to define a html-page and redirect to there (as an error handler does). The problem was, that the fail couldn't prevent that way, because it arises in the module edition.xqm. But it should be possible by a little more work

---

Actually, I wouldn't consider this an error (that would return some HTTP error code). 
For me, it's a valid status – the application is running but there's no data to be displayed. That information need to be displayed to the user.

---

I'm with you, but explained it very bad. I don't want to return an error, but use something like the error handler of exist's controller.

---

The easiest way was to reduce redundancy and simplification. Using the templating module is oversized here.